source/extensions/tracers/xray/localized_sampling.cc

```diff
@@ -10,7 +10,8 @@ namespace Extensions {
 namespace Tracers {
 namespace XRay {
 
-constexpr double DefaultRate = 0.5;
+// DefaultRate=0.05 corresponds to 5% sampling rate when no custom rules are applied
```

Comment:
nit: End with a period.

You also can just start with `// Corresponds to...`.

It's not clear why only this variable gets an explanation, but the others do not. The JSON keys are obvious, but DefaultFixedTarget and SamplingFileVersion may as well get an explanation too.

Reply:
Added the comments (explanation) for the other two variables as well. 👍

And for the previous review comment `> Consider appending something like ...` I made the change in the latest commit. For some reason I was not able to directly reply to that review comment.